nais/naiserator.yaml

```diff
@@ -64,3 +66,6 @@ spec:
      - name: {{@key}}
        value: {{this}}
   {{/each}}
+    - name: FLEX_BUCKET_UPLOADER_SCOPE
+      value: 37e6eb95-2bab-432a-93e4-0356667a724e/.default
```

Comment:
denne er vel ikke det samme i q og prod? Og nå heter variabelen FLEX_BUCKET_UPLOADER_AAD_CLIENT_ID . Foreslår kanskje å ta i bruk dette: https://nav-it.slack.com/archives/C01DE3M9YBV/p1613140877068300

Reply:
👌